tests/test_converter.py

```diff
@@ -174,7 +174,28 @@ def test_no_inputs(self):
             Converter()
         self.assertEqual('At least one input must be given.', f'{cm.exception}')
 
-    def test_both_append_dim_and_concat_dim(self):
-        with self.assertRaises(ConverterError) as cm:
+    def test_both_output_append_dim_and_overwrite(self):
+        with self.assertRaises(ConverterError) as e:
             Converter(input_paths='inputs/*.nc', output_overwrite=True, output_append=True)
-        self.assertEqual('Output overwrite and append flags cannot be given both.', f'{cm.exception}')
+        self.assertEqual(('Output overwrite and append '
+                          'flags cannot be given both.',),
+                         e.exception.args)
+
+    def test_invalid_output_metadata(self):
+        self.add_path('my.zarr')
+
+        with self.assertRaises(ConverterError) as e:
+            # noinspection PyTypeChecker
+            Converter(input_paths='inputs/*.nc',
+                      output_metadata=[('comment', 'This dataset is crap.')])
```

Comment:
Please watch your language.

Reply:
I'd prefer something more neutral as filler here too (and for the other "crap"s of course). Apart from anything else, it's distracting when reading or working on the code.

Reply:
Ok, agreed!